src/main/java/frc/robot/FieldVisionInputGenerator.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class FieldVisionInputGenerator {
+
+    private List<FieldVisionInput> fieldVisionInputsList;
+    private PixyCameraConnector pixyCameraConnector;
+
+    public List<FieldVisionInput> getFieldVisionInput(){
+        
+        fieldVisionInputsList = new ArrayList<FieldVisionInput>();
+        pixyCameraConnector = new PixyCameraConnector();
```

Comment:
NO NO NO NO NO NO.  You're doing what i asked you NOT to do! you're storing a pixy camera reference, and then pulling the data from here